charts/mirrorbits-parent/templates/ingress.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "mirrorbits-parent.name" . -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+{{ include "mirrorbits-parents.labels" . | indent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+{{- if .Values.ingress.className }}
+  ingressClassName: {{ .Values.ingress.className }}
+{{- end }}
+{{- if .Values.ingress.tls }}
+  tls:
+  {{- range .Values.ingress.tls }}
+    - hosts:
+      {{- range .hosts }}
+        - {{ . | quote }}
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
+{{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+    - host: {{ .host | quote }}
+      http:
+        paths:
+        {{- range .paths }}
+          - path: {{ .path }}
+            pathType: {{ .pathType | default "Prefix" }}
+            backend:
+            {{- if eq .backendService "mirrorbits" }}
+              {{- if not (index $.Values "mirrorbits-lite" "enabled") }}
+                {{- fail "Cannot use mirrorbits(-lite) as backend if it is disabled." }}
+              {{- end }}
+              service:
+                # TODO: update child chart to use a global value to let the parent-chart defined/get the value
+                name: {{ $.Release.Name | trunc 63 | trimSuffix "-" -}}-mirrorbits-lite
+                port:
+                  number: {{ index $.Values "mirrorbits-lite" "service" "port" }}
+            {{- else if eq .backendService "httpd" }}
+              {{- if not $.Values.httpd.enabled }}
```

Comment:
I'm really not fond of this `index` stuff required by the dash in the subchart name.

Looking forward to mirrorbits (old) chart deprecation so we can rename mirrorbits-lite and get rid of this incongruity.